ytfzf

```diff
@@ -89,12 +89,16 @@ pid_file="$cache_dir/.pid"
 [ -d "$thumb_dir" ] || mkdir -p "$thumb_dir"
 
 #> config settings
-#list of shortcuts to use in fzf
-#handle_custom_shortcuts will be called if defined (handle_custom_shortcuts can call handle_shortcuts to handle normal shortcuts)
-shortcuts="${shortcuts-alt-l,alt-t,alt-o,alt-v,alt-d,alt-m,alt-s,alt-enter}"
+#handle_custom_key_binds will be called if defined (handle_custom_key_binds can call handle_key_binds to handle normal key binds)
+# key_binds sets the bindings to use for fzf
+# any undefined binds will be used to select
+key_binds="${key_binds-alt-l,alt-t,alt-o,alt-v,alt-d,alt-m,alt-s,alt-enter,enter,double-click}"
```

Comment:
this would cause backward compatibility problems as `$shortcuts` are set in conf.sh. So the new `key_binds` should use that instead

Reply:
yes, we were going to add a warning to change it, `type shortcuts && printf "Error...."`

Reply:
do you want me to add a depreciation warning?